neard/src/cli.rs

```diff
@@ -27,54 +27,96 @@ pub(super) struct NeardCmd {
 impl NeardCmd {
     pub(super) fn parse_and_run() {
         let neard_cmd = Self::parse();
-        neard_cmd.opts.init();
-        info!(target: "neard", "Version: {}, Build: {}, Latest Protocol: {}", crate::NEARD_VERSION, crate::NEARD_BUILD, near_primitives::version::PROTOCOL_VERSION);
-
-        #[cfg(feature = "test_features")]
-        {
-            error!("THIS IS A NODE COMPILED WITH ADVERSARIAL BEHAVIORS. DO NOT USE IN PRODUCTION.");
-
-            if env::var("ADVERSARY_CONSENT").unwrap_or_default() != "1" {
-                error!("To run a node with adversarial behavior enabled give your consent by setting variable:");
-                error!("ADVERSARY_CONSENT=1");
-                std::process::exit(1);
+        let verbose = neard_cmd.opts.verbose.as_deref();
+
+        let rust_log = env::var("RUST_LOG");
+        let rust_log = rust_log.as_ref().map(String::as_str).unwrap_or(near_o11y::DEFAULT_RUST_LOG);
```

Comment:
```suggestion
        let rust_log = rust_log.as_deref().unwrap_or(near_o11y::DEFAULT_RUST_LOG);
```

Reply:
It also feels like maybe we need a function in o11y crate to just return effective value of RusT_LOG?

Reply:
Or maybe even default envfiler